satpy/composites/__init__.py

```diff
@@ -1082,12 +1103,22 @@ def _get_and_sharpen_rgb_data_arrays_and_meta(self, datasets, optional_datasets)
             if 'rows_per_scan' in high_res.attrs:
                 new_attrs.setdefault('rows_per_scan', high_res.attrs['rows_per_scan'])
             new_attrs.setdefault('resolution', high_res.attrs['resolution'])
-            low_res_colors = ['red', 'green', 'blue']
-            low_resolution_index = low_res_colors.index(self.high_resolution_color)
+            colors = ['red', 'green', 'blue']
+            low_resolution_index = colors.index(self.high_resolution_color)
+            high_resolution_index = low_resolution_index
+            if self.neutral_resolution_color is not None:
+                neutral_resolution_index = colors.index(self.neutral_resolution_color)
+                neutral_res = datasets[neutral_resolution_index]
+            else:
+                neutral_res = None
+                neutral_resolution_index = 0
```

Comment:
Is there a way to clean this up? What if `None` was added to the `colors` list? That way you could just do:

```python
neutral_resolution_index = colors.index(self.neutral_resolution_color)
neature_res = datasets[neutral_resolution_index] if neutral_resolution_index is not None else None
```

Not sure if that actually cleans anything up...but yeah.